docs/source/teams/management_sdk.rst

```diff
@@ -0,0 +1,78 @@
+.. _teams-management-sdk:
+
+Teams Management SDK
+===========================
+
+.. default-role:: code
+
+Being able to do the same things in the UI as in Python code is
+one of the big reasons we love FiftyOne!
+
+There are specific operations that only exist for FiftyOne Teams,
+such as management functions for: users, organization invitations,
+dataset permissions, plugins, and API keys.
+
+Setup
+___________________
+In order to use the FiftyOne Teams management SDK, you'll have to
+configure your Python environment to connect to the API endpoint of
+your Teams deployment. To do this, the following configuration
+options are required. See :ref:`Configuring FiftyOne <configuring-fiftyone>`
+for more about the FiftyOne `config` object.
+
++-------------------------------+-------------------------------------+-------------------------------+----------------------------------------------------------------------------------------+
+| Config field                  | Environment variable                | Default value                 | Description                                                                            |
++===============================+=====================================+===============================+========================================================================================+
+| `api_uri`                     | `FIFTYONE_API_URI`                  | `None`                        | The URI where the FiftyOne Teams API (`teams-api` container) is exposed. Check         |
+|                               |                                     |                               | with your Teams admin for the value of this field for your deployment.                 |
++-------------------------------+-------------------------------------+-------------------------------+----------------------------------------------------------------------------------------+
+| `api_key`                     | `FIFTYONE_API_KEY`                  | `None`                        | The FiftyOne Teams API key to use for authentication with the API. This key is         |
+|                               |                                     |                               | unique to each user. See <blah> for information on creating an API key.                |
++-------------------------------+-------------------------------------+-------------------------------+----------------------------------------------------------------------------------------+
+
+
+.. _teams-sdk-api-reference:
```

Comment:
So unfortunately the docs-build CI workflow isn't currently going to have access to a `fiftyone-teams` instance to populate these with.

As a temporary workaround, how about splitting the Teams SDK reference build into a separate script that outputs the `.rst` to stdout/tmp file:

```shell
bash docs/scripts/generate_teams_docs.bash
```

Then, for now, whenever we want to update this section, we run that script and commit the output here?

Reply:
Yeah I feared as much.

The problem with that is that the `.. automodule::` is evaluated at build time so what's here _is_ the `.rst`. We would have to commit the HTML file `build/...management_sdk.html` to this repo

Reply:
Will take offline